Makefile

```diff
@@ -57,23 +57,26 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 endif
 
 ##@ Build Dependencies
+
 ## Tool Binaries
-KUSTOMIZE ?= $(LOCALBIN)/kustomize-$(KUSTOMIZE_VERSION)
-CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen-$(CONTROLLER_TOOLS_VERSION)
-ENVTEST ?= $(LOCALBIN)/setup-envtest-$(ENVTEST_VERSION)
+KUSTOMIZE ?= $(LOCALBIN)/kustomize
+CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
+ENVTEST ?= $(LOCALBIN)/setup-envtest
 OPERATOR_SDK ?= $(LOCALBIN)/operator-sdk
-GOLANGCI_LINT ?= $(LOCALBIN)/golangci-lint-$(GOLANGCI_LINT_VERSION)
+GOLANGCI_LINT ?= $(LOCALBIN)/golangci-lint
 CRD_REF_DOCS ?= $(LOCALBIN)/crd-ref-docs
 YQ ?= $(LOCALBIN)/yq-$(YQ_VERSION)
+
 ## Tool Versions
-KUSTOMIZE_VERSION ?= v5.3.0
+KUSTOMIZE_VERSION ?= v5.4.3
 CONTROLLER_TOOLS_VERSION ?= v0.16.1
-OPERATOR_SDK_VERSION ?= v1.37.0
+OPERATOR_SDK_VERSION ?= v1.39.2
```

Comment:
These changes were implemented in accordance with the upgrade guidelines for Operator SDK v1.38.0 and v1.39.0, as detailed in the official documentation:

- [v1.38.0 Upgrade Guidelines](https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.38.0/)
- [v1.39.0 Upgrade Guidelines](https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.39.0/)